judge/version_check.sh

```diff
@@ -66,6 +66,7 @@ touch version_check.out version_check.meta
 if [ -e "$WORKDIR/version_check-script" ]; then
     mv "$WORKDIR/version_check-script" "$WORKDIR/version_check-script-old"
 fi
+# shellcheck disable=SC2174
 mkdir -m 0777 -p "$WORKDIR/version_check-script"
```

Comment:
I think making the behaviour explicit is better see: https://www.shellcheck.net/wiki/SC2174